change/@internal-react-components-dfd490b8-84bc-430e-bf0f-e32b4877904a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
This type should be `prerelease` since it changes production behavior of a prerelease feature.